src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -1148,7 +1156,7 @@ private boolean populateRecommendation(Map.Entry<String, Terms> termEntry,
         // Check for thresholds
         if (isRecommendedCPURequestAvailable) {
             if (isCurrentCPURequestAvailable && currentCpuRequestValue > 0.0 && null != generatedCpuRequest) {
-                double diffCpuRequestPercentage = CommonUtils.getPercentage(generatedCpuRequest.doubleValue(), currentCpuRequestValue);
```

Comment:
Are we losing any precision by removing `doubleValue()` here?

Reply:
No, we don't.
Java compiler automatically does this unboxing implicitly so writing this `doubleValue()` is actually redundant.